requirements.txt

```diff
@@ -1,15 +1,14 @@
-requests
-elasticsearch>=7.0.0,<8.0.0
+Flask-Caching
+authlib
+cryptography<=3.3.2
```

Comment:
removed the comment describing the issue with this module `https://github.com/pyca/cryptography/issues/5799`

if this was intentional let me know and i'll add it

Reply:
A comment like that should be listed in the git commit message. That is what git is for.